ansible/roles/ooni-backend/tasks/populate_bouncer_base.yml

```diff
@@ -15,13 +15,32 @@
     - "{{ bind_collector_hsdir }}"
     - "{{ bind_web_connectivity_hsdir }}"
 
-- name: Template "{{ bind_bouncer_file  }}"
+- name: Template "{{ bind_bouncer_file }}" with bouncer_base.yaml.j2
   template: >
     src="bouncer_base.yaml.j2"
     dest="{{ bind_bouncer_file }}"
     backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is defined and oonibackend_bundle == True )
+  register: bouncer_config_file
 
-- name: Fetch backend related config files and HS keys
+- name: Template "{{ bind_bouncer_file }}" with "{{ bouncer_base_template }}"
+  template: >
+    src="{{ bouncer_base_template }}"
+    dest="{{ bind_bouncer_file }}"
+    backup="yes"
+    unsafe_writes="true"
+  when: ( oonibackend_bundle is undefined or oonibackend_bundle == False )
```

Comment:
@hellais is there any case where the conditional for `oonibackend_bundle` variable in https://github.com/TheTorProject/ooni-sysadmin/pull/79#discussion-diff-94381954R33 fails to evaluate when `oonibackend_bundle` is unset or set to: Yes/yes, No/no, True/true, False/false ?
If not I will prefer to use the same conditional syntax as in most (if not all) ansible roles/playbooks in this repository.